src/cases/resolvers/activity.resolver.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { ActivityService } from '@hmcts/ccd-case-ui-toolkit';
+import { ActivityResolver } from './activity.resolver';
+
+describe('ActivityResolver', () => {
+
+  let activityService: any;
+  let activityResolver: ActivityResolver;
+
+  beforeEach(() => {
+    activityService = jasmine.createSpyObj<ActivityService>('activityService', ['verifyUserIsAuthorized']);
+    activityService.verifyUserIsAuthorized.and.returnValue(true);
+
+    activityResolver = new ActivityResolver(activityService);
+  });
+
+  it('should verify if user is authorised for activity on resolve and return true when is authorised', () => {
+    let result: boolean;
+    activityResolver.resolve().subscribe(r => result = r);
+    expect(activityService.verifyUserIsAuthorized).toHaveBeenCalled();
+    expect(result).toBe(true);
+  });
+
+  it('should verify if user is authorised for activity on resolve and return true when is not authorised', () => {
```

Comment:
Does this test actually make sense? It should say they're _authorised_ for activity when _not_ authorised? 🤨

Reply:
Most of this stuff was copied over from CCD-UI. I'll have look at the test.

Reply:
Given the code, the test is correct... but seems a bit pointless because `result` cannot ever be anything _other_ than `true`. 😂